mpv.spec

```diff
@@ -95,15 +95,16 @@ waf configure \
     --disable-build-date \
     --enable-libmpv-shared \
     --enable-gpl3 \
-    --enable-sdl2
+    --enable-sdl2 \
+    --enable-encoding
```

Comment:
I bet the patch you've linked to will be in a mpv release sooner than this API really goes away.